ac/ac-autocode/src/index.js

```diff
@@ -42,7 +42,7 @@ export default ({
   configUI,
   validateConfig,
   ActivityRunner,
-  dashboard,
+  dashboards,
```

Comment:
Good to have fixed that grammar mistake. It was bothering me too. Doing a quick text search I find many places in the code where this "s" has not been added. For example in real-life mode I find no dashboard because the test `!!aT.dashboard` returns false

Reply:
Also, I don't think you've changed the type definition

Reply:
fixed